site/docs/spark.md

```diff
@@ -38,12 +38,25 @@ Iceberg uses Spark's DataSourceV2 API for data source and catalog implementation
     Spark 2.4 can't create Iceberg tables with DDL, instead use the [Iceberg API](../api-quickstart).
 
 
-## Spark 2.4
+## Apache Spark 2.4
 
 To use Iceberg in Spark 2.4, add the `iceberg-spark-runtime` Jar to Spark's `jars` folder.
 
 Spark 2.4 is limited to reading and writing existing Iceberg tables. Use the [Iceberg API](../api) to create Iceberg tables.
 
+Recommended way is to include Iceberg's latest released using the `--packages` option:
```

Comment:
I wasn't aware that this works. I think the recommended way to use Iceberg is to use iceberg-spark-runtime, not iceberg-spark. Does that work with `--packages`?

Reply:
Eagle eye @rdblue 

With `org.apache.iceberg:iceberg-spark:0.7.0-incubating`, I get some Guava error:
```
$ spark-shell --packages org.apache.iceberg:iceberg-spark:0.7.0-incubating

Welcome to
    ____              __
   / __/__  ___ _____/ /__
  _\ \/ _ \/ _ `/ __/  '_/
 /___/ .__/\_,_/_/ /_/\_\   version 2.4.4
    /_/
       
Using Scala version 2.11.12 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_172)
Type in expressions to have them evaluated.
Type :help for more information.

scala> import org.apache.spark.sql.SaveMode
import org.apache.spark.sql.SaveMode

scala> val df = spark.read.format("csv").option("header", "true").load("/Users/fokkodriesprong/Desktop/presentations/delta-and-others/data/beers.csv")
df: org.apache.spark.sql.DataFrame = [_c0: string, abv: string ... 6 more fields]

scala> df.write.format("iceberg").mode(SaveMode.Overwrite).save("/tmp/fokko/")
java.lang.NoSuchMethodError: com.google.common.base.Preconditions.checkArgument(ZLjava/lang/String;Ljava/lang/Object;)V
at org.apache.iceberg.spark.source.IcebergSource.createWriter(IcebergSource.java:77)
at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:255)
at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:229)
... 49 elided
```

And with runtime:
```
$ spark-shell --packages org.apache.iceberg:iceberg-spark-runtime:0.7.0-incubating

Welcome to
      ____              __
     / __/__  ___ _____/ /__
    _\ \/ _ \/ _ `/ __/  '_/
   /___/ .__/\_,_/_/ /_/\_\   version 2.4.4
      /_/
         
Using Scala version 2.11.12 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_172)
Type in expressions to have them evaluated.
Type :help for more information.

scala> import org.apache.spark.sql.SaveMode
import org.apache.spark.sql.SaveMode

scala> val df = spark.read.format("csv").option("header", "true").load("/Users/fokkodriesprong/Desktop/presentations/delta-and-others/data/beers.csv")
df: org.apache.spark.sql.DataFrame = [_c0: string, abv: string ... 6 more fields]

scala> df.write.format("iceberg").mode(SaveMode.Overwrite).save("/tmp/fokko/")

scala> spark.read.format("iceberg").load("/tmp/fokko/").count()
res2: Long = 2410
```